wordpress/wp-content/plugins/memberful-wp/src/widgets.php

```diff
@@ -124,7 +124,7 @@ function memberful_wp_format_widget_links( $links ) {
     $links[$key] = '<a href="'.$link['href'].'" class="'.$link['class'].'">'.$link['text'].'</a>';
   }
 
-  return implode( ' | ', $links );
+  return implode( ' <span class="memberful-links-separator">|</span> ', $links );
```

Comment:
This class isn't defined anywhere in the plugin? Is this intentional and do we expect our users to define it if they want to change things?